### ISSUE TITLE
Fund transfer missing

### ISSUE THREAD

**Issue by DamienPoh** [kucoin-community-chain/any-advice-issue#21]
Sent a withdrawal request from Kucoin to Blockfi through KCC but both parties claim it is not retrievable. Please help
![Screenshot_20210831-025646](https://user-images.githubusercontent.com/89806146/131391277-028b1722-e0a1-4970-9cb8-26a25d9d53a2.png)

**Comment by 1218016476** [comment]
You REALLY should remove your attachment. ANYONE that knows how to retrieve the $$ can do it and it's gone!

**Comment by ghost** [comment]
Same thing happened to me a few days ago. Lost over 35 million Shiba Inu. And no one will help. Good luck to you. Something needs to be done to fix this.

**Comment by S52Hertz** [comment]
me too just transferred it but i didn't receive my money yet

**Comment by ghost** [comment]
Yeah I get it and it really sucks. It took me months of small purchases of Shiba Inu to get to 35 million +. And since coinbase put it on there exchange I wanted to transfer my Sheba from kucoin to coinbase and it just went poof. I am literally sick about it. From everything I read we will never retrieve the funds lost. Coinbase would have to add the k c c network. And even then they may be gone forever.

Sent from Yahoo Mail on Android 
 
  On Sun, Oct 24, 2021 at 2:27 PM, ***@***.***> wrote:   


me too just transferred it but i didn't receive my money yet

—
You are receiving this because you commented.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
